hawc/main/settings/base.py

```diff
@@ -123,9 +123,6 @@
 if os.getenv("HAWC_INCLUDE_ECO", "False") == "True":
     INSTALLED_APPS = INSTALLED_APPS + ("hawc.apps.eco",)
 
-if os.getenv("HAWC_INCLUDE_EPIV2", "False") == "True":
```

Comment:
Unfortunately, because the yaml file now has epiv2 models in it, the test database can't load unless epiv2 is installed so the unit tests always fail

Reply:
duh, that makes sense, my bad. ok to remove; we have this on a separate branch so we're good.

Is this PR ready to review?